test/converters/junitxml/junitxml.go

```diff
@@ -34,15 +72,15 @@ func parseTestSuites(filePath string) ([]junit.TestSuite, error) {
 	var testSuites junit.XML
 	testSuitesError := xml.Unmarshal(data, &testSuites)
 	if testSuitesError == nil {
-		return testSuites.TestSuites, nil
+		return regroupErrors(testSuites.TestSuites), nil
 	}
 
 	var testSuite junit.TestSuite
 	if err := xml.Unmarshal(data, &testSuite); err != nil {
 		return nil, errors.Wrap(errors.Wrap(err, string(data)), testSuitesError.Error())
```

Comment:
why do you include the `testSuitesError` in the `parseTestSuites ` function's error message?
i feel `parseTestSuites` error should only describe why test suit parsing failed.

Reply:
because of the fallback, the first error message might can tell something